projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5364,19 +5369,19 @@ public function add_token_to_login_redirect_json_api_authorization( $redirect_to
 	 */
 	public function verify_json_api_authorization_request( $environment = null ) {
 		$environment = $environment === null
-			? $_REQUEST
+			? $_REQUEST // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- would appreciate a double check here.
```

Comment:
This function looks like it's doing its own security checking, including its own nonces (look around line 5449).

Reply:
2dc7e0d8c1a9999981b1cd3b2d05279a6b94fdbb

Reply:
Looks ok to me